### PR TITLE
see all items in checklist details

### PR DESCRIPTION
**Pull request by helenrendich**

### CODE REVIEW

File: frontend/src/models/Checklist.ts [hunk 1]
```diff
@@ -1,7 +1,10 @@
...
 export type Checklist = {
     id: string
     destination: string
     startDate: string
+    items: Item[]
 }
 
-export type NewChecklist = Omit<Checklist, "id">
\ No newline at end of file
+export type NewChecklist = Omit<Checklist, "id" | "items">
```
Comment (hamideh6182) [inline frontend/src/models/Checklist.ts:10]: Maybe you can define it so, then you don't need to Omit :
`id? : string  
destination: string
startDate: string 
items?: Item[]`

**Review by hamideh6182 — APPROVED** [review]
Good job 👍

**Comment by helenrendich** [comment]
> Please have a look at your checklist when u update and adventure 😉 is it still in progress, or is it a bug?

Both, kind of. It was a forgotten todo that needed to be done. It should be fixed now.

**Review by hamideh6182 — APPROVED** [review]
👍

**Review by HendrikNeuschulz — APPROVED** [review]

**Pull request merged**